src/toil/common.py

```diff
@@ -729,60 +914,65 @@ def addOptions(parser: ArgumentParser, config: Optional[Config] = None, jobstore
                          'or a specific model like nvidia-tesla-k80), and a count [default: 1]. If both a type and a count '
                          'are used, they must be separated by a colon. If multiple types of accelerators are '
                          'used, the specifications are separated by commas')
-    resource_options.add_argument('--defaultMemory', dest='defaultMemory', default=None, metavar='INT',
-                                  help=resource_help_msg.format('default', 'memory', disk_mem_note, bytes2human(config.defaultMemory)))
-    resource_options.add_argument('--defaultCores', dest='defaultCores', default=None, metavar='FLOAT',
-                                  help=resource_help_msg.format('default', 'cpu', cpu_note, str(config.defaultCores)))
-    resource_options.add_argument('--defaultDisk', dest='defaultDisk', default=None, metavar='INT',
-                                  help=resource_help_msg.format('default', 'disk', disk_mem_note, bytes2human(config.defaultDisk)))
-    resource_options.add_argument('--defaultAccelerators', dest='defaultAccelerators', default=None, metavar='ACCELERATOR[,ACCELERATOR...]',
-                                  help=resource_help_msg.format('default', 'accelerators', accelerators_note, config.defaultAccelerators))
-    resource_options.add_argument('--defaultPreemptible', '--defaultPreemptable', dest='defaultPreemptible', metavar='BOOL',
-                                  type=bool, nargs='?', const=True, default=False,
+
+    h2b = lambda x: human2bytes(str(x))
+
+    resource_options.add_argument("--default_memory", '--defaultMemory', dest='defaultMemory', default=2147483648, type=h2b, action=make_int_range_validation_action(1),
```

Comment:
Shouldn't these magic numbers also be `SYS_MAX_SIZE`?

Reply:
It seems like `defaultMemory` and `defaultDisk` controls the amount of resources that the system will request, and setting it to `SYS_MAX_SIZE` will return InsufficientResources. The docs say defaultMemory should be 2GB and the value here is 2GiB, so maybe the default should be changed to match the docs?